internal/testcontext/compile_norace.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build !race
```

Comment:
What does this do?

Reply:
That is a build tag. 
* https://golang.org/pkg/go/build/#hdr-Build_Constraints
* https://dave.cheney.net/tag/build-tags

That `race` constraint is specifically for `-race`.